.changeset/chilly-ants-grin.md

```diff
@@ -0,0 +1,12 @@
+---
+"@aws-amplify/ui-react": major
+---
+
+BREAKING CHANGE: `isMultiline` prop removed from `TextField`. Replace any `TextField` components 
+using the `isMultiline` prop with the `TextAreaField` component.
+
+```diff
+- <TextField
+-  isMultiline
++ <TextAreaField
+```
```

Comment:
Maybe we should include a complete example? That makes the change logs look better

Reply:
What kind of example are you thinking? Assuming the props are on multiple lines, wouldn't this be the diff?

Reply:
Yeah, that should be fine. I was thinking TextArea is a new primitive but i realized we have get it out for a while.